src/views/CartView.vue

```diff
@@ -55,10 +64,10 @@
                 role="tabpanel"
               >
                 <b-card-body class="mx-3 my-1">
-                  <div v-for="subsection in section.subsections" :key="subsection.id">
+                  <div v-for="(subsection, _index) in section.subsections" :key="subsection.id">
                     <h5 class="h6">{{subsection.name}}</h5>
                     <ul>
-                      <li v-for="variable in variablesWithSet(subsection.variables)" :key="variable.id" class="subvariable-line" :class="variableSetClass(variable)">
+                      <li v-for="(variable, __index) in variablesWithSet(subsection.variables)" :key="`${index}-${_index}-${__index}`" class="subvariable-line" :class="variableSetClass(variable)">
```

Comment:
index, __index and __index, can there be renamed to more descriptive names

Reply:
Updated naming to indicate the current loop (sectionIndex, subsectionIndex, variableIndex)